listenbrainz/webserver/views/feedback_api.py

```diff
@@ -0,0 +1,74 @@
+import ujson
+import listenbrainz.db.user as db_user
+import listenbrainz.db.feedback as db_feedback
+
+from datetime import datetime
+from flask import Blueprint, current_app, jsonify, request
+from listenbrainz.webserver.decorators import crossdomain
+from listenbrainz.webserver.errors import (APIBadRequest,
+                                           APIInternalServerError, APINotFound,
+                                           APIServiceUnavailable,
+                                           APIUnauthorized)
+from listenbrainz.webserver.rate_limiter import ratelimit
+from listenbrainz.webserver.views.api import _validate_auth_header
+from listenbrainz.webserver.views.api_tools import log_raise_400, DEFAULT_ITEMS_PER_GET, MAX_ITEMS_PER_GET
+from listenbrainz.feedback import Feedback
+from pydantic import ValidationError
+
+#: Maximum overall feedback size in bytes, to prevent egregious spamming.
+MAX_FEEDBACK_SIZE = 10240
```

Comment:
if we even keep this, we should drastically reduce this to something near the actual size.

Reply:
Removed